### PR TITLE
add google site verification tag

### PR DESCRIPTION
**Pull request by Dhaulagiri**
### :pushpin: Summary

This will allow us to have the site indexed by google. I piggybacked this on the analytics addon since they work the same, but I'm open to feedback on that.

### :link: External links

* [Verify your site ownership](https://support.google.com/webmasters/answer/9008080)

### CODE REVIEW

File: website/lib/analytics/index.js [hunk 1]
```diff
@@ -5,7 +5,9 @@ module.exports = {
 
   contentFor(type) {
     if (type === 'head-footer' && process.env.VERCEL_ENV === 'production') {
-      return `<script src="https://cdn.usefathom.com/script.js" data-spa="auto" data-site="PPQLJJKK" data-excluded-domains="localhost" defer></script>`;
+      return `<script src="https://cdn.usefathom.com/script.js" data-spa="auto" data-site="PPQLJJKK" data-excluded-domains="localhost" defer></script>
```
Comment (alex-ju) [inline website/lib/analytics/index.js:8]: Do we also want to exclude the vercel preview deploys?
```suggestion
      return `<script src="https://cdn.usefathom.com/script.js" data-spa="auto" data-site="PPQLJJKK" data-excluded-domains="vercel.app,localhost" defer></script>
```

**Review by alex-ju — APPROVED** [review]

File: website/lib/analytics/index.js (hunk 1 above)
Comment (Dhaulagiri) [reply website/lib/analytics/index.js:8]: `process.env.VERCEL_ENV === 'production'` will exclude those and https://hds-website.vercel.app/ is now redirected to our main domain.

Comment (alex-ju) [reply website/lib/analytics/index.js:8]: ah, nice!

Comment (Dhaulagiri) [reply website/lib/analytics/index.js]: can't hurt though 😄

**Pull request merged**